hypervisor/fanout.go

```diff
@@ -62,12 +60,11 @@ func (fo *Fanout) Release(client chan *types.VmResponse) error {
 	for _, c := range fo.clients {
 		if c != client {
 			remains = append(remains, c)
-		} else {
-			fo.unregistered = append(fo.unregistered, c)
+			continue
 		}
+		close(client)
```

Comment:
it closes remained channel.
please move it into the else branch and remove the continue